packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -46,6 +46,30 @@ async function setup(): Promise<Fixture<Table<SimpleTableRecord>>> {
     );
 }
 
+async function growTableToGivenRowWidth(
+    rowWidth: number,
+    table: Table<SimpleTableRecord>
+): Promise<void> {
+    if (!table.$fastController.isConnected) {
+        throw Error(
+            "You must call 'await connect()' before calling this method"
+        );
+    }
+
+    const collapseButton = table.shadowRoot?.querySelector(
+        '.collapse-all-button'
+    );
+    const buttonWidth = collapseButton?.getBoundingClientRect().width;
+    const buttonStyle = window.getComputedStyle(collapseButton!);
+    table.style.width = `${
+        rowWidth
+        + buttonWidth!
```

Comment:
nit: If you're going to make a not-null assertion, I think it would be better suited on `collapseButton` rather than on `buttonWidth`. It took me a minute to realize how `buttonWidth` could be undefined -- by `collapseButton` being undefined. Therefore, I think it would be cleaner to assert the button exists so that its width isn't possibly undefined.